test/universal.js

```diff
@@ -449,6 +449,17 @@ describe('Google Analytics :: Universal', function() {
       });
     });
 
+    describe('#refundedOrderPartial', function() {
+      it('should send the right data', function(done) {
+        var json = test.fixture('refunded-order-partial');
+        test
+          .set(settings)
+          .track(json.input)
+          .sendsAlmost(json.output, {ignored: ['qt']})
```

Comment:
Pretty sure I wrote the `sendsAlmost` test to ignore `qt` for this reason. IIRC it's because it varies depending on exactly how fast the test runs.